solutions/remotemonitoring/single-vm/setup.sh

```diff
@@ -45,14 +44,21 @@ while [ "$#" -gt 0 ]; do
         --aad-instance)                 PCS_WEBUI_AUTH_AAD_INSTANCE="$2" ;;
         --release-version)              PCS_RELEASE_VERSION="$2" ;;
         --docker-tag)                   PCS_DOCKER_TAG="$2" ;;
-        --evenhub-connstring)           PCS_EVENTHUB_CONNSTRING="$2" ;;
+        --eventhub-connstring)          PCS_EVENTHUB_CONNSTRING="$2" ;;
```

Comment:
I didn't check but his typo is fixed in arm template as well?

Reply:
yes.